generator/README.md

```diff
@@ -124,7 +124,7 @@ Some of these are quite sluggish, so use wget to download.
 Put the extracted mibs in a location NetSNMP can read them from. `$HOME/.snmp/mibs` is one option.
 
 * Cisco: ftp://ftp.cisco.com/pub/mibs/v2/v2.tar.gz
-* APC: ftp://ftp.apc.com/apc/public/software/pnetmib/mib/421/powernet421.mib
+* APC: [PowerNet-MIB](https://download.schneider-electric.com/files?p_File_Name=powernet426.mib)
```

Comment:
The raw url should be left here, for easy copy&pasting.

Reply:
Ok, done.